AppVeyor fails building

Due to recent update of AppVeyor VS2019 image that upgraded MSBuild `( 16.4.0+e901037fe)->(16.5.0+d4cbfca49)`, the builds now fail (`error MSB3030: Could not copy the file "C:\projects\gameserver\GameServerConsole\obj\Debug\netcoreapp3.0\GameServerConsole.exe" because it was not found. `. It should be bin/Debug instead of obj...)

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -13,6 +13,7 @@ install:
       $env:BUILD = "LeagueSandboxServer-v{0}.7z" -f $env:COMM_HASH
   - git submodule update --init --recursive
 
+# Set to previous due to build errors for copying GameServer
 image: Previous Visual Studio 2019
 
 build:

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -13,7 +13,7 @@ install:
       $env:BUILD = "LeagueSandboxServer-v{0}.7z" -f $env:COMM_HASH
   - git submodule update --init --recursive
 
-image: Visual Studio 2019
+image: Previous Visual Studio 2019
 
 build:
   project: GameServer.sln

